ask-smapi-model/CHANGELOG.rst

```diff
@@ -222,3 +222,11 @@ This release contains the following changes :
 This release contains the following changes :
 
 - Updating model definitions for `App link interfaces <https://developer.amazon.com/en-US/docs/alexa/alexa-for-apps/skill-manifest-reference.html>`__.
+
+1.14.1
+^^^^^^
+
+This release contains the following changes :
+
+- Enable TSB APL Extension in skill manifest
+- Remove INTERNAL scope for A/B testing endpoint
```

Comment:
Changelog is for external and visible to 3P customers.
INTERNAL scope is not relevant to models, since its related to our autogenerating of models from specifications.
Don't add these in future